config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java

```diff
@@ -30,11 +30,14 @@
  */
 public final class ConfigurationFactory {
     private static final Logger LOGGER = LoggerFactory.getLogger(ConfigurationFactory.class);
-    private static final String REGISTRY_CONF = "registry.conf";
+    private static final String REGISTRY_CONF_PREFIX = "registry";
+    private static final String REGISTRY_CONF_SUFFIX = ".conf";
     /**
      * The constant FILE_INSTANCE.
      */
-    public static final Configuration FILE_INSTANCE = new FileConfiguration(REGISTRY_CONF);
+    private static final String ENV_VALUE = System.getProperty("env");
```

Comment:
maybe you can provide a util class to set or resolve the env value
default it can fetch from system properties
when users use spring ，maybe they want to config from the application.properties ，so they can invoke your util method  instead of  use `System.setProperty` or pass by `-D`